content/guides/5.collections.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Collections and Playlists
+slug: collections
+fullpath: /guides/collections
+---
+
+---
+
+# Introduction
+This page describes the similarities and differences between a collection and playlist. Collections and podcasts are another way to organize individual books and podcast episodes (not podcasts).
+
+Both Collections and Playlists:
+- Are Library specific
+- Listening progress is tracked by user (not shared listening state between multiple users)
+- Have a description field
+- Can be reordered
+- Can contain podcast episodes
```

Comment:
Currently collections can only contain books. We may expand this in the future but so far there hasn't been a need.

Reply:
Yep, you're correct. I mixed up which list was which.